Day12/.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
can you keep all the code inside himanshu/ folder, so that when we merge other code, there wont be any conflicts